metrics/network/nuttcp_dockerfile/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Usage: FROM [image name]
+FROM ubuntu
+
+# Version of the Dockerfile
+LABEL DOCKERFILE_VERSION="1.0"
+
+# Version of nuttcp
+ARG NUTTCP_VERSION="7.3.2.c"
```

Comment:
Ah - I thought this looked a bit odd :) Please can you make this:

```
ARG NUTTCP_VERSION="7.3.2"
  :

gcc nuttcp-${NUTTCP_VERSION}.c -o nuttcp
```

I still worry that we have two code locations with the same values - it's going to be difficult to ensure they stay in sync I think.